pom.xml

```diff
@@ -3,9 +3,14 @@
      xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>net.snowflake</groupId>
+  <parent>
+    <groupId>net.snowflake</groupId>
+    <artifactId>snowflake-jdbc-base</artifactId>
+    <version>3.14.3</version>
+    <relativePath>base_pom.xml</relativePath>
```

Comment:
do we need to add ./ as it's relative path?

Reply:
It works for me locally but it doesn't hurt to add ./ in the path. I'll add it since I need to resolve conflicts again.